runbotics/runbotics-scheduler/src/database/database.module.ts

```diff
@@ -11,10 +10,11 @@ import { ProcessInstanceEventModule } from './process-instance-event/process-ins
 import { ScheduleProcessModule } from './schedule-process/schedule-process.module';
 import { BotCollectionModule } from './bot-collection/bot-collection.module';
 import { BotSystemModule } from './bot-system/bot-system.module';
-import { FeatureKeyModule } from './feature-key/feature-key.module';
 import { TriggerEventModule } from './trigger-event/trigger-event.module';
 import { ProcessInstanceLoopEventModule } from './process-instance-loop-event/process-instance-loop-event.module';
 import { GuestModule } from './guest/guest.module';
+import { AuthorityModule } from '#/scheduler-database/authority/authority.module';
+import { FeatureKeyModule } from '#/scheduler-database/feature-key/feature-key.module';
```

Comment:
I think you could move those entities into scheduler-database module. I know that is not in the same schema, but I think it will be clear if we store directly new migrations in that file. The old one then could be refactored/deleted. This is a small thing, we could discuss it if u want.

Reply:
Alright, done